### PR TITLE
Fix ValueError

### PR DESCRIPTION
**Pull request by Pavocracy**
Fixes issue #70

### CODE REVIEW

File: stockfish/models.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __init__(
             path, universal_newlines=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE
         )
 
-        self._stockfish_major_version: int = int(self._read_line().split(" ")[1])
+        self._stockfish_major_version: float = float(self._read_line().split(" ")[1])
```
Comment (zhelyabuzhsky) [inline stockfish/models.py:39]: `_stockfish_major_version` should stay `int`

If stockfish version is `14.1`, this variable should contain `int` value `14`

Comment (zhelyabuzhsky) [reply stockfish/models.py:39]: please add test for this case

**Review by zhelyabuzhsky — CHANGES_REQUESTED** [review]
Thanks for your contribution )

File: stockfish/models.py (hunk 1 above)
Comment (Pavocracy) [reply stockfish/models.py]: Fixed

Comment (zhelyabuzhsky) [reply stockfish/models.py]: Well done

**Comment by zhelyabuzhsky** [comment]
@Pavocracy fix lint problems pls and I will merge your PR )

**Comment by Pavocracy** [comment]
@zhelyabuzhsky Fixed. Sorry to have to commit twice on a PR. I should have checked the build.yml the first time to see what build steps I should have run locally to test first. Will keep this in mind for the future :)

**Review by zhelyabuzhsky — APPROVED** [review]

**Pull request merged**